99_master-node-sizing-enabled-env.yaml.in

```diff
@@ -0,0 +1,17 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-node-sizing-for-crc
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,${DYNAMIC_DATA}
```

Comment:
It seems this data can be inline? (looking at https://github.com/harche/machine-config-operator/blob/4fdea851cf8984722518895ed7b07b746ff09fa1/templates/common/_base/files/kubelet-auto-sizing.yaml )
```

contents:
  inline: |
    NODE_SIZING_ENABLED=false
    SYSTEM_RESERVED_MEMORY=350Mi
    SYSTEM_RESERVED_CPU=200m